src/Diagnostics/FxCop/VisualBasic/Design/CodeFixes/CA1052BasicCodeFixProvider.vb

```diff
@@ -15,13 +15,17 @@ Imports Microsoft.CodeAnalysis.VisualBasic
 
 Namespace Design
 
-    <ExportCodeFixProvider(LanguageNames.VisualBasic, Name:=StaticTypeRulesDiagnosticAnalyzer.RuleNameForExportAttribute), [Shared]>
+    <ExportCodeFixProvider(LanguageNames.VisualBasic, Name:=CA1052DiagnosticAnalyzer.DiagnosticId), [Shared]>
     Public Class CA1052BasicCodeFixProvider
         Inherits CodeFixProvider
 
         Public NotOverridable Overrides ReadOnly Property FixableDiagnosticIds As ImmutableArray(Of String)
             Get
-                Return ImmutableArray.Create(StaticTypeRulesDiagnosticAnalyzer.CA1052RuleId)
+                ' TODO: Re-implement the VB fix by turning the Class into a Module.
```

Comment:
Can you please create an issue for this so that we don't lose track (and reference the issue in code here).